contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -56,6 +62,8 @@ class ExpectColumnValuesToBePresentInAnotherTable(QueryExpectation):
         foreign_table_key_column: key column in foreign table.
     """
 
+    MAX_UNEXPECTED_VALUES_TO_RENDER: Final[int] = 20
```

Comment:
Should this be a private attr? It’ll pop up in autocomplete if a user is instantiating this Expectation due to how our Pydantic params work

Reply:
If anything, let's annotate with `ClassVar` so we're clear

Reply:
After discussion, let's pull this from result_format (which also defaults to 20).

Reply:
After further discussion, we don't have access to result_format. Will is going to investigate.